frontend/src/__tests__/cypress/cypress/e2e/pipelines/Pipelines.cy.ts

```diff
@@ -354,6 +354,7 @@ describe('Pipelines', () => {
     pipelinesGlobal.selectPipelineServerAction('Delete pipeline server');
     deleteModal.findSubmitButton().should('be.disabled');
     deleteModal.findInput().fill('Test Project pipeline server');
+    deleteModal.findSubmitButton().should('have.text', 'Delete pipeline server');
```

Comment:
We do not want tests to be asserting static text as it has a 1:1 relationship with the code and will never fail unless the static text changes.